evennia/commands/default/building.py

```diff
@@ -1455,6 +1455,11 @@ class CmdSetAttribute(ObjManipCommand):
 
     Switch:
         edit: Open the line editor (string values only)
+        script: If we're trying to set an attribute on a script
+        channel: If we're trying to set an attribute on a channel
+        account: If we're trying to set an attribute on an account
+        (room, exit, char/character may all be used as well for global
```

Comment:
This should probably be phrased differently - rom, exit, char are valid switches but putting them in parentheses makes that unclear.